src/Feed/LoginFeed.ts

```diff
@@ -108,6 +126,11 @@ export default function useLoginFeed() {
         dispatch(addNotifications(notifications));
     }, [notificationFeed.store]);
 
+    useEffect(() => {
+      const ps = getMutedKeys(mutedFeed.store.notes)
+      dispatch(setMuted(ps))
+    }, [mutedFeed.store])
+
     useEffect(() => {
         let dms = dmsFeed.store.notes.filter(a => a.kind === EventKind.DirectMessage);
         dispatch(addDirectMessage(dms));
```

Comment:
I've decided against it since we don't always have `muted` list available by the time we start getting dm events. Doing filtering on the downstream components instead.

Reply:
Good point